src/svg/qsvggenerator.cpp

```diff
@@ -166,6 +169,8 @@ class QSvgPaintEngine : public QPaintEngine
                        svgEngineFeatures())
     {
     }
+	int clip_counter = 0;
```

Comment:
why do you need `clip_counter` at all? Why is it public? Is that meant to be used outside of this class?